tailwind.config.js

```diff
@@ -16,6 +19,7 @@ module.exports = {
         secondary: '#F4DCF4',
         'app-background': '#fff',
         orange: '#ff5a42',
+        blue: '#389bb7',
```

Comment:
rename blue to 'date-header' as we might use green instead of blue for the header tomorrow